sources/packages/backend/apps/api/src/route-controllers/student/student.institutions.controller.ts

```diff
@@ -68,4 +70,26 @@ export class StudentInstitutionsController extends BaseController {
   ): Promise<StudentProfileAPIOutDTO> {
     return this.studentControllerService.getStudentProfile(studentId);
   }
+
+  /**
+   * This controller returns all student documents uploaded by student uploader.
+   * @param studentId student id.
+   * @returns list of student documents.
+   */
+  @Get(":studentId/documents")
+  async getInstitutionStudentFiles(
+    @Param("studentId", ParseIntPipe) studentId: number,
+  ): Promise<InstitutionStudentFileAPIOutDTO[]> {
+    const studentDocuments = await this.fileService.getStudentUploadedFiles(
```

Comment:
since it is exactly the same as the AEST controller, we can move the common code to the shared controller service and call the shared service 
![image](https://user-images.githubusercontent.com/77353155/233132714-21ea91b8-2106-4a01-b17b-dad8eb6a0e58.png). Maybe you can refer to the profile controller https://vscode.dev/github/bcgov/SIMS/blob/a2442286f936c0ee7621fa39a105aaa36607fe36/sources/packages/backend/apps/api/src/route-controllers/student/student.institutions.controller.ts#L68

Reply:
Yes we can move this code to controller service and could be used for student as well.